go/ephemeral/lib.go

```diff
@@ -388,6 +388,20 @@ func (e *EKLib) getOrCreateLatestTeamEKInner(ctx context.Context, teamID keybase
 	return teamEK, nil
 }
 
+// Try to get the TeamEK for the given `generation`. If this fails and the
+// `generation` is also the current maxGeneration, create a new teamEK.
+func (e *EKLib) GetMaybeCreateNewTeamEK(ctx context.Context, teamID keybase1.TeamID, generation keybase1.EkGeneration) (teamEK keybase1.TeamEk, err error) {
```

Comment:
Can we change the name somehow to clarify that even if the function creates a new key, it won't return it?

Reply:
I was kinda stuck on a name -- what are your thoughts? I thought this was different enough from `GetOrCreateLatest` to be clear

Reply:
Maybe we should just call it `GetTeamEK` since the create is just an internal detail for a better experience. Side note we should think about notifying the frontend when we're creating teamEKs since it might be slow for large teams

Reply:
Actually that reminds me of another important bit: If it takes (say) 10 seconds to create a teamEK, we have to make sure that multiple attempts to send an exploding message don't result in us creating multiple teamEKs.